src/bug-filing/common/http-query-builder.ts

```diff
@@ -9,7 +9,8 @@ export type QueryParam = {
 };
 
 export class HTTPQueryBuilder {
-    public static readonly maxUrlLength = 3000;
+    // defacto max lenght for urls
```

Comment:
Is defacto a Spanish word?

Reply:
latin.. is actually de facto.. I should fix that